backend/dal/Migrations/v01.07.02/Up/PostUp/Adminstrative_Areas_And_Addresses.sql

```diff
@@ -0,0 +1,191 @@
+-- Update addresses table first to correct locations which contain District of, City of, or The Corporation of the District of
+PRINT 'Updating Addresses table'
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'District of ', '')
+where AdministrativeArea like 'District of %';
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'City of ', '')
+where AdministrativeArea like 'City of %';
+
+UPDATE dbo.[Addresses]
+SET AdministrativeArea = REPLACE(AdministrativeArea, 'The Corporation of the District of ', '')
+where AdministrativeArea like 'The Corporation of the District of %';
+
+-- Do the "one off" updates to simplify the name where there is only once instance of each
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Spallumcheen' 
+WHERE AdministrativeArea = 'The Corporation of the Township of Spallumcheen';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Lake Country' 
+WHERE AdministrativeArea = 'District Of Lake Country';
```

Comment:
It would be yes, I had just copied and pasted my "one-offs" without reviewing the script. There are quite a few redundant updates that I can get rid of.